README.md

```diff
@@ -1 +1,55 @@
-# create-nuget-package-action
\ No newline at end of file
+# create-nuget-package-action
+
+<img alt= "" src="https://github.com/OctopusDeploy/create-nuget-package-action/raw/main/assets/github-actions-octopus.png" />
```

Comment:
I'm struggling to see this image, did we copy it across? We use the same image for all of the actions, so just copy the assets folder from one of the others.

Reply:
yeah, we missed it